examples/basic_openshift/custom-values.yaml

```diff
@@ -0,0 +1,15 @@
+authenticators: "authn-k8s/minikube,authn"
+dataKey: "iMldjotGK9DcKUjKfCv+k6vzVSoL/6ODTpS0CAoKio8="
+account: "myaccount"
```

Comment:
I think it would be great if there would be one place to define the constants that are used in more than one place. For example, the `account` name is currently hard coded in multiple places. If someone wants to make a change, it will not be straight forward to change it.